pages/freebsd/sed.md

```diff
@@ -0,0 +1,29 @@
+# sed
+
+> Edit text in a scriptable manner.
+> See also: `awk`, `ed`.
+> More information: <https://www.freebsd.org/cgi/man.cgi?sed>.
+
+- Replace all `apple` (basic regex) occurrences with `mango` (basic regex) in all input lines and print the result to `stdout`:
+
+`{{command}} | sed 's/apple/mango/g'`
+
+- Execute a specific script [f]ile and print the result to `stdout`:
+
+`{{command}} | sed -f {{path/to/script.sed}}`
+
+- Delay opening each file until a command containing the related w function or flag is applied to a line of input:
```

Comment:
```suggestion
- Delay opening each file until a command containing the related function or flag is applied to a line of input:
```

Reply:
This is not a typo, should we use "`w` function instead"?

Reply:
Oh, yeah wrapping it in backticks would be much better.